README.md

```diff
@@ -11,25 +12,38 @@ ITKElastix
 Overview
 --------
 
-Provides an [ITK](https://www.itk.org) Python interface to [elastix](https://elastix.lumc.nl/), a toolbox for rigid and nonrigid registration of images.
+Provides an [ITK](https://www.itk.org) Python, JavaScript, and WebAssembly interface to [elastix](https://elastix.lumc.nl/), a toolbox for rigid and nonrigid registration of images.
 
 elastix is open source software, based on the well-known [Insight Toolkit (ITK)](https://discourse.itk.org). The software consists of a collection of algorithms that are commonly used to solve (medical) image registration problems. The modular design of elastix allows the user to quickly configure, test, and compare different registration methods for a specific application.
 
+[👨‍💻 **Live JavaScript API Demo** ✨](https://itk-wasm-elastix-app-js.on.fleek.co/ ':include :type=iframe width=100% height=800px')
+
 Installation
 ------------
 
-Install cross-platform binary Python packages with [pip](https://pypi.org/project/pip/):
+Install cross-platform native binary Python packages with [pip](https://pypi.org/project/pip/):
+
+```sh
+pip install itk-elastix
+```
+
+*Experimental* WebAssembly packages can be installed across platforms with:
 
-    pip install itk-elastix
+```sh
+pip install itkwasm-elastix
+```
 
-*Experimental* GPU-accelerated packages can be installed on Linux with:
+JavaScript / TypesScript packages cas be installed with
+
+```sh
+npm install @itk-wasm/elastix
+```
 
-    pip install itk-elastix-opencl
 
 Usage
 -----
 
-To register two images, traditionally called the fixed image and the moving image:
+To register two images with the native Python binaries, traditionally called the fixed image and the moving image:
```

Comment:
Do the WebAssembly Python packages have a different Python interface than native binaries?

Reply:
They are slightly different. And they will change somewhat in the future. Currently an initial ITK Transform file or TransformParameterObject can be passed, and both an output ITK Transform file and a TransformParameterObject are produced. However, in the future, the ITK Transforms will be in-memory objects.

Reply:
Thanks. Could it please be mentioned somewhere in this README that minor differences exist?

Reply:
Good idea, done.